storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1200,13 +1200,21 @@ bool Redo_Log_Data_Manager::stop_at(lsn_t lsn, lsn_t checkpoint_lsn) {
 
   scanned_lsn = reader.get_scanned_lsn();
 
+  ut_ad(scanned_lsn >= stop_lsn);
```

Comment:
I think the assertion is wrong. Please check
May be use ut_ad(scanned_lsn == stop_lsn);